holo-vrrp/src/packet.rs

```diff
@@ -384,6 +386,7 @@ impl Ipv4Hdr {
 }
 
 impl EthernetHdr {
+    const LEN: usize = 14;
```

Comment:
nit: please add a newline after this line to separate the constant from the function

Reply:
done.